### DISCUSSION TITLE
Using Hantek with USB to fiber Optic Extender

### DISCUSSION THREAD

**Discussion by anatoltrush** [OpenHantek/OpenHantek6022#343] [Q&A] [answered]
Has someone used Hantek with USB to fiber Optic Extender?
When I try to connect Hantek to Extender (via optic cable) and Extender to PC (via USB) application (Hantek 6000) gets stuck.

**Comment by Ho-Ro** [comment, marked as answer]
Due to the simple Hantek HW without buffer memory it needs an USB Hi-Speed 480 Mb/s connection to transfer the samples in real time - also the use of USB isolators is difficult.
